packages/neon-core/src/tx/components/TransactionAttribute.ts

```diff
@@ -63,8 +74,9 @@ export class TransactionAttribute {
       throw new Error(`Data size too big!`);

```

Comment:
This error doesnt hold anymore if you are storing it as ASCII instead of hex. The original design was always to store hex.